package com.qa.javaException10Hrs;

public class Case1_Try_Catch_finally {
	public static void main(String[] args) {

		try {
			System.out.println("try");
		} catch (Exception e) {
			System.out.println("catch");
		} finally {
			System.out.println("Finally");
		}

	}
}
